src/core/mac/#mac.cpp#

```diff
@@ -0,0 +1,2112 @@
+/*
```

Comment:
what's the purpose of this file?

Reply:
occasionally generated invalid file, deleted